dotnet/src/IntegrationTests/Agents/OpenAIAssistantAgentTests.cs

```diff
@@ -216,6 +219,56 @@ await OpenAIAssistantAgent.CreateAsync(
         finally
         {
             await agent.DeleteThreadAsync(threadId);
+            await agent.DeleteAsync();
+        }
+    }
+
+    /// <summary>
+    /// Integration test for <see cref="OpenAIAssistantAgent"/> using function calling
+    /// and targeting Open AI services.
+    /// </summary>
+    [Fact]
+    public async Task AzureOpenAIAssistantAgentStreamingFileSearchAsync()
+    {
+        var azureOpenAIConfiguration = this._configuration.GetSection("AzureOpenAI").Get<AzureOpenAIConfiguration>();
+        Assert.NotNull(azureOpenAIConfiguration);
+
+        OpenAIClientProvider provider = OpenAIClientProvider.ForAzureOpenAI(new AzureCliCredential(), new Uri(azureOpenAIConfiguration.Endpoint));
+        OpenAIAssistantAgent agent =
+            await OpenAIAssistantAgent.CreateAsync(
+                provider,
+                new(azureOpenAIConfiguration.ChatDeploymentName!),
+                new Kernel());
+
+        // Upload file - Using a table of fictional employees.
+        OpenAIFileClient fileClient = provider.Client.GetOpenAIFileClient();
+        await using Stream stream = File.OpenRead("TestData/employees.pdf")!;
```

Comment:
just wondering if this works well since it is not added in the project def or if does it need something like in processes samples: `GetRepositoryProcessStateFilepath`

Reply:
Yes, it does.  Its included by this section in the .csproj:

https://github.com/microsoft/semantic-kernel/blob/main/dotnet/src/IntegrationTests/IntegrationTests.csproj#L198

Unlike samples, the file is simply copied and not an embedded resource.